djstripe/settings.py

```diff
@@ -150,6 +151,23 @@ def get_subscriber_model():
     return subscriber_model
 
 
+def get_api_version():
+    """Get the desired API version to use for Stripe requests."""
+    version = getattr(settings, 'DJSTRIPE_API_VERSION', None) or '2017-02-14'
```

Comment:
Following the `STRIPE_LIVE_MODE` and `STRIPE_*_*_KEY` settings, I would say name it `STRIPE_API_VERSION`.